examples/basic_candlepin.pp

```diff
@@ -33,7 +33,6 @@
   command => "/usr/bin/keytool -import -v -keystore ${truststore} -alias candlepin-ca -file ${ca_cert} -noprompt -storepass ${truststore_password} -storetype pkcs12",
   creates => $truststore,
 } ->
-group { 'tomcat': } ->
```

Comment:
Nice! This should probably have been our hint that we should have managed it a long time ago.